File: Position.cpp
//  Noam shimshoviz 203565429 Sarit Zevin 313242588
#include <iostream>
#include "Position.h"

Position::Position(int x, int y): row(x), column(y) {}

const int& Position::getColumn() const {

    return column;
}

const int& Position::getRow() const {
    return row;
}

bool Position::operator ==(const Position& other) {

    return ((row == other.getRow()) and (column == other.getColumn()));
}
ostream &operator <<(ostream &out, const Position &point) {
    out << " (" << point.getRow() << ", " << point.getColumn() << ") ";
    return out;
}

File: javaFXEx6/Position.java
package javaFXEx6;
//Noam shimshoviz 203565429 Sarit Zevin 313242588

public class Position {
	private int row;
	private int column;
	public Position(int r, int c){
		this.row = r;
		this.column = c;
	}
	
	public int getColumn() {
	    return column;
	}
	public int getRow() {
	    return row;
	}
	public boolean equal(Position other){
		if(this.getColumn()==other.getColumn() && this.getRow() == other.getRow()){
			return true;
		}
		return false;
	}
}
